src/show/pages/public/Home/index.jsx

```diff
@@ -39,82 +37,32 @@ const Home = () => {
           </div>
         </div>
       </div>
-      <div className='flex bg-white h-60 overflow-x-auto'>
-        <div className='flex ml-5 mt-3 fixed  text-black text-2xl font-bold antialiased'>
+      <div className='flex flex-col bg-white h-60 overflow-x-auto'>
+        <div className='flex ml-5 mt-3 text-black text-2xl font-bold antialiased'>
           Constructors
         </div>
-        <div className='flex ml-5 mt-10 flex-row justify-between'>
-          <ConstructorCard src='http://assets.stickpng.com/images/580b585b2edbce24c47b2c52.png' />
-          <ConstructorCard src='https://logodownload.org/wp-content/uploads/2014/04/mercedes-benz-logo-1-1.png' />
-          <ConstructorCard src='https://brandlogos.net/wp-content/uploads/2022/05/red_bull_racing-logo-brandlogos.net_.png' />
-          <ConstructorCard src='https://upload.wikimedia.org/wikipedia/commons/thumb/7/7e/Alpine_F1_Team_Logo.svg/2233px-Alpine_F1_Team_Logo.svg.png' />
-          <ConstructorCard src='https://upload.wikimedia.org/wikipedia/en/thumb/6/66/McLaren_Racing_logo.svg/2560px-McLaren_Racing_logo.svg.png' />
-          <ConstructorCard src='https://upload.wikimedia.org/wikipedia/commons/thumb/3/3f/Logo_Alfa_Romeo_Racing_Orlen.svg/2560px-Logo_Alfa_Romeo_Racing_Orlen.svg.png' />
-          <ConstructorCard src='https://cdn.worldvectorlogo.com/logos/scuderia-alpha-tauri-1.svg ' />
-          <ConstructorCard src='https://www.gran-turismo.com/gtsport/decal/8574862557608674816_1.png' />
-          <ConstructorCard src='https://upload.wikimedia.org/wikipedia/commons/d/d4/Logo_Haas_F1.png' />
-          <ConstructorCard src='https://i.imgur.com/ckOGBzR.png' />
+        <div className='flex ml-5 mt-3 flex-row'>
+          {constructors?.map(constructors => (
+            <ConstructorCard key={constructors.id} constructor={constructors} />
+          ))}
```

Comment:
You map over an array of constructors, so each one of them is called `constructor` not `constructors`.